client/packages/common/src/api/GqlContext.tsx

```diff
@@ -170,9 +163,9 @@ export const GqlProvider: FC<PropsWithChildren<ApiProviderProps>> = ({
   skipRequest,
   children,
 }) => {
-  const [{ client }, setApi] = useState<{
-    client: GQLClient;
-  }>(() => createGql(url, skipRequest));
+  const client = useRef(
+    new GQLClient(url, { credentials: 'include' }, skipRequest)
+  ).current;
```

Comment:
Seems like a good case for `useRef` rather than `useState`, as we're not going to be "setting state" anymore, we're just going to mutate `client` directly using its internal setters.